roles/cloud-bootstrap/tasks/create-users-projects.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: Create bonnyci project
+  os_project:
+    cloud: "{{ cloud }}"
+    name: bonnyci
+
+- name: Create bonnyci group
+  os_group:
+    cloud: "{{ cloud }}"
+    name: bonnyci
+
+- name: Create bonnyci-admin group
+  os_group:
+    cloud: "{{ cloud }}"
+    name: bonnyci-admin
+
+- name: Add all users to bonnyci group
+  os_user_group:
+    cloud: "{{ cloud }}"
+    group: bonnyci
+    user: "{{ item.username }}"
+  with_items: users
+
+- name: Add all bonnyci group members to bonnyci project
+  os_user_role:
+    cloud: "{{ cloud }}"
+    role: member
+    group: bonnyci
+    project: bonnyci
+
+- name: Add admin users to bonnyci-admin group
+  os_user_group:
+    cloud: "{{ cloud }}"
+    group: bonnyci-admin
+    user: "{{ item.username }}"
+  with_items: users
+  when: item.isadmin == True
+
+- name: Grant bonnyci-admin group cloud_admin on bonnyci project
+  os_user_role:
+    cloud: "{{ cloud }}"
+    role: cloud_admin
+    group: bonnyci-admin
+    project: bonnyci
+
+- name: Create nodepool project
+  os_project:
+    cloud: "{{ cloud }}"
+    name: nodepool
+
+- name: Create nodepool user
+  os_user:
+    cloud: "{{ cloud }}"
+    name: nodepool
+    default_project: nodepool
```

Comment:
@gandelman-a I used `secrets.nodepool.db_password` since I didn't see just a `password` var in the example secrets.